### PR TITLE
Add UI elements for Events

### PR DESCRIPTION
**Pull request by fusin02**

### CODE REVIEW

**Comment by fusin02** [comment]
closes #86

**Comment by fusin02** [comment]
closes #87

File: src/main/java/seedu/address/ui/EventCard.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+     * Creates a {@code EventCard} with the given {@code Event} and index to display.
+     */
+    public EventCard(Event event, int displayedIndex) {
+        super(FXML);
+        this.event = event;
+        id.setText(displayedIndex + ". ");
+        eventName.setText(event.getName().getEventName());
+        time.setText(event.getTime().getTime());
+        venue.setText(event.getVenue().getVenue());
+        celebrity.getChildren().add(new Label(event.getCelebrity().getName().fullName));
```
Comment (wrjgold) [inline src/main/java/seedu/address/ui/EventCard.java:50]: Should we apply SOLID principles here?

**Review by wrjgold — COMMENTED** [review]
Generally good, just want to understand reason for setting Edit,Delete commands to isShowPersonList = false and HelpCommand isShowPersonList = true.

**Review by wrjgold — APPROVED** [review]
lgtm

**Pull request merged**